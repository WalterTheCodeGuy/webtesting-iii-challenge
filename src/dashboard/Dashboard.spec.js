  
import React from 'react'
import '@testing-library/jest-dom/extend-expect'
import Dashboard from './Dashboard'
import { render, fireEvent } from '@testing-library/react'

test('Renders dashboard correctly', ()=> {
    const wrapper = render(<Dashboard/>);
    expect(wrapper.asFragment()).toMatchSnapshot();
})