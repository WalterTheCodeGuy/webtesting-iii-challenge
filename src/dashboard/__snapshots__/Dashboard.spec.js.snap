// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Dashboard /> should match snapshot 1`] = `
Array [
  <div
    className="display panel"
  >
    <div
      className="led green-led"
    >
      Unlocked
    </div>
    <div
      className="led green-led"
    >
      Open
    </div>
  </div>,
  <div
    className="controls panel"
  >
    <button
      className="toggle-btn"
      disabled={true}
      onClick={[Function]}
    >
      Lock Gate
    </button>
    <button
      className="toggle-btn"
      disabled={false}
      onClick={[Function]}
    >
      Close Gate
    </button>
  </div>,
]
`;

exports[`Renders dashboard correctly 1`] = `
<DocumentFragment>
  <div
    class="display panel"
  >
    <div
      class="led green-led"
    >
      Unlocked
    </div>
    <div
      class="led green-led"
    >
      Open
    </div>
  </div>
  <div
    class="controls panel"
  >
    <button
      class="toggle-btn"
      disabled=""
    >
      Lock Gate
    </button>
    <button
      class="toggle-btn"
    >
      Close Gate
    </button>
  </div>
</DocumentFragment>
`;
